coreneuron/utils/progressbar/progressbar.c

```diff
@@ -32,6 +32,18 @@ enum { WHITESPACE_LENGTH = 2 };
 /// The amount of width taken up by the border of the bar component.
 enum { BAR_BORDER_WIDTH = 2 };
 
+/// Progress Bar max number of updates
+enum { BAR_UPDATES_NUMBER = 500 };
+
+/// Progress Bar interval of updates (seconds)
+enum { BAR_UPDATES_INTERVAL = 5 };
+
+/// Progress Bar time of change of updates (seconds)
+enum { BAR_UPDATES_CHANGE = 120 };
```

Comment:
more suggestive names: 
```
/// The reference bar redraw interval
BAR_DRAW_INTERVAL = 5    // maybe (isatty()? 1 : 5)

/// The maximum number of updates (avoid frequent output on very long runs ) 
BAR_DRAW_COUNT_MAX = 1000

/// The Initial period, where BAR_DRAW_INTERVAL is always respected
BAR_DRAW_FREQUENT_PERIOD = 120

Reply:
Thank you very much. I will take care of this and try to be more descriptive the next time 👍

Reply:
@ferdonline Please check the next commit and give me some feedback if you want on the variables' names descriptiveness.